### DISCUSSION TITLE
Issue with the matrix writer after reinstallation of latest Bonsai version

### DISCUSSION THREAD

**Discussion by benefron** [bonsai-rx/bonsai#1550] [General]
Attached is a simplified workflow to show the issue I encountered.
After uninstalling and reinstalling the latest version  2.8.0 (2.8.0.4248) of Bonsai, I ran my regular workflow, the filenames from the matrix writer exploded. My workflow creates a new file based on certain events in my workflow. I update the file name with a suffix based on the built-in property of Suffix in the matrixWriter node. When I run it now, instead of taking the file name and adding the suffix, it adds a an additional suffix for each new file. for example, test0.bin, test01.bin, test011.bin, test.0111bin etc. Or if you try for a timestamp suffix you get a file name that quickly explodes, for example: tt_other2023-09-10T13_38_342023-09-10T13_38_362023-09-10T13_38_372023-09-10T13_38_382023-09-10T13_38_392023-09-10T13_38_40.bin
In my uploaded workflow, I control the new file creation in two ways: a continuous loop and another controlled by key presses. In both ways, I encounter the same issue. 
[tes_matrixWritert.zip](https://github.com/bonsai-rx/bonsai/files/12568017/tes_matrixWritert.zip)

**Comment by glopesdev** [comment]
@benefron thank you for this, we have identified the problem and is indeed a new issue which we will aim to fix as soon as possible, we will track its progress here: https://github.com/bonsai-rx/bonsai/issues/1551

**Reply by glopesdev** [reply]
@benefron this is now fixed, merged and published via #1554. If you update your system package to the latest version 2.8.1 hopefully this issue will be gone.
